areas/models.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+from munigeo.utils import get_default_srid
+
+PROJECTION_SRID = get_default_srid()
+
+
+class ContractZone(models.Model):
+    name = models.CharField(verbose_name=_('name'), max_length=255)
```

Comment:
Do we need translations for those or no?

Reply:
No idea, so I opted for the simpler solution for now :)